README.md

```diff
@@ -23,9 +23,65 @@ command line:
 poetry install
 ```
 
+```bash
+poetry install typer
+```
```

Comment:
This command is an invalid command, and the intention behind it should never be needed -- `poetry install` will install all dependencies listed in `pyproject.toml`, which includes typer. There is no need to install it separately.